src/models/implementations/presentable-model.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Nectary\Models;
+
+use Nectary\Model;
+
+/**
+ * A Viewable model is one that can present itself so that all of its
+ * properties and calculated properties are useable within a template
+ */
+abstract class Presentable_Model extends Model {
```

Comment:
@rraub Still trying to think of a better term for this. I think Presentable_Model is a little more flexible than Viewable_Model, especially if you need to call `present` to get the results from it.